### ISSUE TITLE
How to reproduce the accuracy value for nnUNet pytorch model

### ISSUE THREAD

**Issue by arunjose696** [NVIDIA/DeepLearningExamples#961]
I was trying to reproduce the results for nnunet(https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/Segmentation/nnUNet#training-accuracy-results) model on a V100 gpu. I run the training script I am able to see the loss logged to standard out 

Epoch 0: 100%|██████████████████████████████████████████████████████████████| 291/291 [04:16<00:00,  1.13it/s, loss=1.221]DLL 2021-07-05 09:26:56.923081 - Epoch: 0  mean dice : 46.27  TOP_mean : 46.27  L1 : 47.92  L2 : 40.51  L3 : 50.37  TOP_L1 : 47.92  TOP_L2 : 40.51  TOP_L3 : 50.37  val_loss : 1.2756
Epoch 1: 100%|██████████████████████████████████████████████████████████████| 291/291 [04:06<00:00,  1.18it/s, loss=0.848]DLL 2021-07-05 09:31:03.021547 - Epoch: 1  mean dice : 56.78  TOP_mean : 56.78  L1 : 67.32  L2 : 42.47  L3 : 60.55  TOP_L1 : 67.32  TOP_L2 : 42.47  TOP_L3 : 60.55  val_loss : 0.8893

But could anyone help me with how to get the training accuracy numbers ie Accuracy - FP32, that are present in the table as I am not able to see accuracy metric  in the log, is there any existing script to check the training accuracy, or should we write custom code to get the training accuracy.

**Comment by michal2409** [comment]
The reported accuracy in the README is the TOP_mean score which is the highest dice score reported during training (averaged over five folds, i.e. the TOP_mean scores were gathered from experiments for folds 0, 1, 2, 3, 4 and then averaged).

**Comment by arunjose696** [comment]
Thanks @michal2409   this makes things clear. While I have you could I clear on more doubt. Is the "Time to train"  documented  in the table the time taken to run scripts/train.py script completely or is it the time difference between timestamps for last and first epochs.

**Comment by michal2409** [comment]
Time to train was calculated as follows:

1. run the training for folds 0, 1, 2, 3, 4,
2. for each fold record time to train as the difference between the first timestamp and timestamp with the highest mean dice (the train.py scipts runs with the early stopping policy with patience 100 epochs),
3. average collected time to train.

**Comment by arunjose696** [comment]
thanks michal2409 this was really helpfull

**Comment by michal2409** [comment]
you welcome!
